docs/en/api/POSIX-API.md

```diff
@@ -260,46 +262,61 @@ alluxio.job.worker.threadpool.size=64
-Alluxio FUSE can provide another layer of local metadata/data cache on the application nodes to further speed up the metadata/data access.
+When an application runs an operation against the local FUSE mount point.
+The request will be processed by FUSE kernel, Fuse process, and Alluxio system sequentially.
+If at any level, cache is enabled and cache hit, cached metadata/data will be returned to the application without going through the whole process to improve the overall read performance.
+
+While Alluxio system (master and worker) provides remote distributed metadata/data cache to speed up the metadata/data access of Alluxio under storage files/directories,
+Alluxio FUSE provides another layer of local metadata/data cache on the application nodes to further speed up the metadata/data access.
 
-Alluxio FUSE can provide two kinds of metadata/data cache, the kernel cache and the userspace cache. Kernel cache is executed by Linux kernel
-and metadata/data are stored in operating system kernel cache. Userspace cache is controlled and managed by Alluxio FUSE process
-and metadata/data are stored in user configured location (process memory for metadata, ramdisk/disk for data).
+Alluxio FUSE can provide two kinds of metadata/data cache, the kernel cache and the userspace cache.
+Kernel cache is executed by Linux kernel with metadata/data stored in operating system kernel cache.
+Userspace cache is controlled and managed by Alluxio FUSE process with metadata/data stored in user configured location (process memory for metadata, ramdisk/disk for data).
+Although both kinds of cache can be enabled at the same time, it’s recommended to choose only one of them to avoid double memory consumption.
 The following illustration shows the layers of cache — FUSE kernel cache, FUSE userspace cache, Alluxio system cache.
 
 <p align="center">
 <img src="{{ '/img/posix-cache.png' | relativize_url }}" alt="Alluxio stack with its POSIX API"/>
 </p>
 
-Alluxio FUSE cache is a single-node cache solution which means modifications through other Alluxio clients or other Alluxio FUSE mount points
-may not be visible immediately by the current Alluxio FUSE cache. The cached data may be stale.
-For example, an application on `Node A` may delete `file` and rewrite `file` with new content
-when an application on `Node B` is reading the `file` from its local FUSE cache,
-the content read is stale.
+Alluxio FUSE cache (Userspace cache or Kernel cache) is a single-node cache solution,
+which means modifications to the underlying Alluxio cluster through other Alluxio clients or other Alluxio FUSE mount points
+may not be visible immediately by the current Alluxio FUSE cache. This would cause cached data to become stale.
+For example, `Node A` may read a cached file without knowing that Node B had already deleted or overwritten the file in the underlying Alluxio cluster.
+When this happens the content read by `Node A` is stale.
```

Comment:
since this covers both metadata cache and data cache, and you had to mention them in both places. what if we put this as a subsection parallel to them ? #### it should stand out, and u dont have to mention this in 3 places?

Reply:
## Cache Tuning
### Single Node Cache
### Metadata Cache
### Data Cache
Users may feel confusing adding a new subsection, may make the other two feels like not single node cache solution